setup.py

```diff
@@ -57,7 +49,14 @@
             "pytest-regressions",
             "beautifulsoup4",
         ],
-        "rtd": ["sphinxcontrib-bibtex", "ipython"],
+        "rtd": [
```

Comment:
@choldgraf I moved the `pandas_sphinx_theme` dependency to the `rtd` extra, because its not a hard dependency of using sphinx.